### PR TITLE
Adding 'shapes' to the note

### LINKED ISSUE #2972: 2.5.8: Target offset differ not only if the sizes of the targets differ
[definition of target offset](https://www.w3.org/TR/WCAG22/#dfn-target-offsets):

> The target offset from A to B may be different than the target offset from B to A, if the sizes of these targets differ.

Even if the size is the same, but the objects are rotated, target offset can be different, as the example shows

258b
Figure 1: Two identical triangles, one standing on its baseline, the other on a corner. Two arrows show target offset, which is different 

Even if they are not rotated, but positioned offset, elements of the same size can have a different target offset

258a
Figure 2: Two identical triangles, aligned horizontally (according to the definition of target offset), but not at the same height. Two arrows show target offset, which is different

### PR DESCRIPTION
**Pull request by alastc**
Closes #2972

### CODE REVIEW

File: guidelines/terms/22/target-offset.html [hunk 1]
```diff
@@ -3,5 +3,5 @@
 	<p class="change">New</p>	
    <p>length of the longest possible line that starts at an edge of a target (A), intersects a second edge of A, and ends at the closest edge of a second target (B). For horizontally aligned targets, target offset is measured with a horizontal line. For vertically aligned targets, target offset is measured with a vertical line. For targets that are neither, target offset is measured diagonally.</p>
    <p>Two targets are horizontally aligned if a horizontal line can be drawn that goes through both targets, but no vertical line can be drawn that goes through both targets. Two targets are vertically aligned if a vertical line can be drawn that goes through both targets, but no horizontal line can be drawn that goes through both targets.</p>
-   <p class="note">The target offset from A to B may be different than the target offset from B to A, if the sizes of these targets differ.</p>
+   <p class="note">The target offset from A to B may be different than the target offset from B to A, if the sizes or shapes of these targets differ.</p>
```
[the remark this answers is not in the document]
Comment (JAWS-test) [reply guidelines/terms/22/target-offset.html:6]: Instead of removing everything after the comma, inserting "for example" would also be sufficient

**Review by detlevhfischer — APPROVED** [review]
"...target offset from A to B may be different than the target offset from B to A"
"different than" sounds odd. I think the better grammer might be:
"...target offset from A to B may differ from the target offset from B to A"

**Comment by alastc** [comment]
Putting that all-together:
"The target offset from A to B may differ from the target offset from B to A, for example, if the sizes or shapes of these targets differ."

File: guidelines/terms/22/target-offset.html (hunk 1 above)
Comment (mbgower) [inline guidelines/terms/22/target-offset.html:6]:
```suggestion
   <p class="note">The target offset from A to B may be different than the target offset from B to A, for example if the sizes or shapes of these targets differ.</p>
```

**Pull request merged**